frontend/src/app/AppContext.ts

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
+import { StorageClassKind } from '~/k8sTypes';
 import { DashboardConfigKind } from '~/k8sTypes';
```

Comment:
interesting, DashboardConfigKind is in main and incubation, but not in f/storage-class ...